pkg/podexecutor/staticpod.go

```diff
@@ -57,6 +57,8 @@ type CloudProviderConfig struct {
 func (s *StaticPodConfig) Kubelet(args []string) error {
 	extraArgs := []string{
 		"--volume-plugin-dir=/var/lib/kubelet/volumeplugins",
+		"--file-check-frequency=5s",
+		"--sync-frequency=30s",
```

Comment:
Seems to shorten the initial static pod startup by a fair bit, without much additional overhead to steady-state operation.

Reply:
Tuned by Rancher!